corehq/apps/hqwebapp/static/hqwebapp/less/_hq/utils.less

```diff
@@ -8,10 +8,6 @@
   display: block;
 }
 
-.sr-only {
-  display: none;
-}
-
```

Comment:
Were you able to tell why this was added in the first place?

Reply:
I checked all the places it was being used in this way, but it didn't seem to be critical at all. Everything that was completely hidden by this class seemed to function as it should with the bootstrap class. @biyeun made the change 6 years ago. Biyeun do you remember this?

Reply:
i'm sure it was important at the time. But 6 years ago...i definitely don't remember